### DISCUSSION TITLE
Sample script for plotting sky localizations

### DISCUSSION THREAD

**Discussion by cwjames1983** [chime-frb-open-data/community#31] [Q&A] [answered]
This is a request for a new feature.

HDF5 files containing localisation data for each FRB, i.e. likelihoods as a function of sky direction, and contours. In theory, the data allows one to reproduce the localisation plots that appear in CHIME/FRB papers. This is extremely useful data for other astronomers to use. However, finding this data, and interpreting it - while not too difficult - is not trivial either (e.g. I have currently lost a link to the page containing this info and cannot find it...).

My suggestion is two-fold:
A: Provide a link to the hdf5 localisation data for each FRB, preferably on the FRB data page e.g. here https://www.chime-frb.ca/catalog/FRB20180725A under "localisation". 
B: Under the "Information" tab, include a sample python3 script to interpret this data and make a localisation plot, such that the produced plot is identical to that shown on the FRB data page. It should be a minimal script that produces that plot and nothing more, i.e. showing how to incorporate all the data contained in the hdf5 file.

The intention of these changes is to greatly improve the usability of the data for other researchers.

## Expected Behavior

For "A": Clicking on the link under 'localisation' should trigger the hdf5 download
For 'B": running the python3 script - assuming basic packages are already installed - should produce a pdf/png/etc file showing the localisation plot.

## Current Behavior

Currently, it is difficult to find the hdf5 data, and I have not been able to determine the coordinates of the likelihood map (I am not sufficiently familiar with the healpix system).

## Possible Solution

I expect that such a script for "B" already exists, and making this publicly available would be very easy. Adding in appropriate data links for "A" could be more difficult.

## Steps to Reproduce (for bugs)

N/A

## Context

I am trying to search for host galaxy associations with CHIME FRBs, which requires being able to associate a likelihood with a given sky coordinate; and in addition, plot any found galaxies on the same likelihood plots that CHIME uses for clarity of interpretation. This is relevant to two current paper drafts (which of course would cite CHIME ;-) )

## Your Environment

* Python Version: 3.7.7
* Environment name and version (e.g. Chrome 39,): Chrome 94.0.4606.71
* Operating System and version (desktop or mobile): macOS Big Sur 11.5.2
* Link to your project: N/A (Although I am part of CRAFT)

Thanks!
-Clancy

**Comment by shinybrar** [comment, marked as answer]
@cwjames1983 Are you alluding to this tutorial page?

https://chime-frb-open-data.github.io/localization/

**Comment by cwjames1983** [comment]
Thanks, I had not found that!
